content/blog/2016-09-02-promcon-2016-recap.md

```diff
@@ -0,0 +1,152 @@
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<blockquote class="twitter-tweet tw-align-center" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/hashtag/promcon2016?src=hash">#promcon2016</a> was amazing. Thanks to everybody involved <a href="https://twitter.com/PrometheusIO">@PrometheusIO</a> <a href="https://twitter.com/juliusvolz">@juliusvolz</a></p>&mdash; tex (@texds) <a href="https://twitter.com/texds/status/769213616541298688">August 26, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Overall, we were very happy with how PromCon turned out - no event is perfect,
+but for a small community conference organized in free time, it exceeded most
+people's expectations.
+
+## Thanks
+
+PromCon 2016 would not have been possible without the help of its sponsors,
+speakers, attendees, and organizers. Thanks so much to all of you! Our Diamond
+and Platinum sponsors deserve a special mention at this point, since they did
+the most to support us and made all the food, drinks, video recordings, and
+swag possible:
+
+<h3>Diamond</h3>
+<div class="sponsor-logos">
+  <a href="http://www.robustperception.io/"><img src="/assets/blog/2016-09-02/robust_perception_logo.png"/></a>
+  <a href="https://www.weave.works/"><img src="/assets/blog/2016-09-02/weave_logo.png"/></a>
+</div>
+
+<h3>Platinum</h3>
+<div class="sponsor-logos">
+  <a href="https://cncf.io/"><img src="/assets/blog/2016-09-02/cncf_logo.png"/></a>
+  <a href="https://coreos.com/"><img src="/assets/blog/2016-09-02/coreos_logo.svg"/></a>
+</div>
+
+We would also like to thank Google for hosting the conference at their office
+in Berlin!
+
+## Outlook
+
+If PromCon 2016 went so well, when will the next one happen?
+
+<blockquote class="twitter-tweet tw-align-center" data-lang="en"><p lang="en" dir="ltr">Promcon was nice. Looking forward to next time. Bye Berlin. <a href="https://twitter.com/hashtag/promcon2016?src=hash">#promcon2016</a> <a href="https://t.co/XInN9OR3pL">pic.twitter.com/XInN9OR3pL</a></p>&mdash; Robert Jacob (@xperimental) <a href="https://twitter.com/xperimental/status/769520813385117697">August 27, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+The answer is that we don't know for sure yet. This first PromCon was organized
+entirely in people's free time, with most of it handled by one person. This
+will surely have to change, especially as we also expect a next Prometheus
+conference to be much larger (even this year, the limited tickets sold out
+within seconds). In the next months, will discuss within the community what we
+want PromCon to be, who should run it, and where it should take place. Perhaps
+there is even space for multiple Prometheus confereces around the world. We will
```

Comment:
What! Normally I *am* the spell checker ;) Thanks, fixed. Also did what you suggested, the rest seems to look good.